jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -3184,10 +3184,6 @@ void java_bytecode_convert_methodt::save_stack_entries(
 {
   for(auto &stack_entry : stack)
   {
-    // remove typecasts if existing
-    while(stack_entry.id()==ID_typecast)
```

Comment:
So this will cause the code below to get hit much less often (looking for symbol, or deref, or member operator at top level) -- what is the upshot of this? Did this code ever have tests? If not this bug surely cries out for some, perhaps requested from @mgudemann as the original author?